crates/extensions/c8y_mapper_ext/src/service_monitor.rs

```diff
@@ -28,7 +28,8 @@ pub fn convert_health_status_message(
         return vec![];
     }
 
-    if entity.topic_id.is_bridge_health_topic() {
+    // the bridge itself is not registered as a service, only the mapper
+    if entity.topic_id == crate::C8Y_BRIDGE_TOPIC_ID {
```

Comment:
With this change, the `az` bridge is reported as a service on `c8y`. This could make sense, but is this what we expect?